fixtures/EN10168/v0.3.0/template.hbs

```diff
@@ -0,0 +1,1135 @@
+<html xmlns="http://www.w3.org/1999/xhtml">
+
```

Comment:
The template has been updated is the schema repo and now uses a partial that should also be added in the fixtures.